config/schema.yaml

```diff
@@ -927,6 +929,105 @@ schema:
         - Tract.skymap
         - Tract.tract
 
+    CalibrationLabel:
+      doc: >
+        A join table that associates an instrument and label with a date range,
+        used to associate master calibration products with exposures.
+      limited: false
+      columns:
+      -
+        name: instrument
+        type: string
+        length: 8
+        primary_key: true
+        nullable: false
+        doc: >
+          Name of the instrument associated with these master calibration
+          products.
+      -
+        name: calibration_label
+        type: string
+        length: 128
+        primary_key: true
+        nullable: false
+        doc: Label used to identify a group of master calibration products.
+      -
+        name: valid_first
+        type: datetime
+        doc: >
+          TAI timestamp of first exposure included in the range (inclusive).
+          May be far in the past to indicate an open interval.
+      -
+        name: valid_last
+        type: datetime
+        doc: >
+          TAI timestamp of last exposure included in the range (inclusive).
```

Comment:
For this ticket I'd like to keep it inclusive to retain the previous logic (or, at least, the old as-documented logic - thinking more about the changes to the unit tests I had to make, I suppose this may actually be a change to the behavior to bring it in line with the documentation).

Anyhow, I think I did have some concrete use-case reasons for why I thought an inclusive upper bound would be better here, and I think it did involve the fact that there are always gap between exposures.  But I'm not sure I can reconstruct all of that thinking right now, and I'm happy to revisit it in the future.